components/Input/TextInput.tsx

```diff
@@ -46,11 +49,17 @@ export function TextInput({
         maxLength={79}
         spellCheck="false"
       />
+      {!!onClear && (
```

Comment:
I moved the clear input button into the component so that we can use it in other places too